components/Tabs/ProjectCard.jsx

```
@@ -31,11 +49,36 @@ export const ProjectCard = ({ project }) => {
+              href={primaryLink.url}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <primaryLink.icon size={16} className="mr-2" />
+              {primaryLink.label}
+              <ArrowUpRight size={14} className="ml-1" />
+            </Link>
+          </Button>
+        ) : (
+          <div></div>
+        )}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)